export function throwError() {
  console.trace('foo')
}
